### DISCUSSION TITLE
QNeat3 (advanced qgis tutorial: shortest path)

### DISCUSSION THREAD

**Discussion by ahorvat89** [ua-gist604b-s24/syllabus#17] [General]
Has anyone been able to run the QNeat3 plugin? I have tried a few times and it just keeps running the OD Matrix from Layers as Line (m:n) algorithm without producing an output until I have to force quit.

**Comment by JMPhotos** [comment]
Works for me, though it took over 4 minutes on a high-end system. If you're an older or slower system, it'll likely take longer. Remember it's calculating 67,000 items, and it looks to like it does the whole thing in multiple steps. Don't trust the progress bar.

First it'll calculate each point (and weirdly, the progress bar stays at 100% the whole time.)
![image](https://github.com/ua-gist604b-s24/syllabus/assets/48196832/df84c5bb-1b75-486c-a5d8-52b5921f868d)

Then it'll process each. Not exactly sure what it entails, but the progress bar resets and shows progress for this bit.
![image](https://github.com/ua-gist604b-s24/syllabus/assets/48196832/c843bdf5-aa08-4729-b7b1-14fa3e3c1463)

**Reply by aaryno** [reply]
Also addressed in lecture - the workaround for this not finishing in a reasonable time is to simply sample fewer points (such as 100).
